docs/kubernetes/user-guides/snapshots.md

```diff
@@ -0,0 +1,51 @@
+# Kubernetes Snapshots User Guide (Alpha)
+
+### FIXME: Install Driver with alpha snapshot feature
+
+1. [One-time per project] Create GCP service account for the CSI driver and set required roles
+```
+$ PROJECT=your-project-here                       # GCP project
+$ GCE_PD_SA_NAME=my-gce-pd-csi-sa                 # Name of the service account to create
+$ GCE_PD_SA_DIR=/my/safe/credentials/directory    # Directory to save the service account key
+$ ./deploy/setup-project.sh
+```
+
+2. Deploy driver to Kubernetes Cluster
+```
+$ GCE_PD_SA_DIR=/my/safe/credentials/directory    # Directory to get the service account key
+$ GCE_PD_DRIVER_VERSION=alpha                     # Driver version to deploy
```

Comment:
are we creating a new "alpha" overlay? Right now I think `dev` basically serves this function(ish)

Reply:
I think we should update the overlay/docs atomically so we don't have a period of time when docs say to deploy alpha but that doesn't actually do anything

Reply:
yes, i want to create a separate alpha overlay that we can use for another CI test job.  I will work on adding one

Reply:
I decided to just put the overlay back to dev for now.  With topology going beta in 1.14, it's only snapshots that is still alpha.